src/sentry/static/sentry/app/views/settings/organization/general/organizationSettingsForm.jsx

```diff
@@ -22,11 +23,17 @@ const NewOrganizationSettingsForm = React.createClass({
     onSave: PropTypes.func.isRequired,
   },
 
-  mixins: [ApiMixin],
+  mixins: [ApiMixin, OrganizationState],
 
   render() {
     let {initialData, orgId, onSave} = this.props;
 
+    //Only for adding the Flag to 2FA Enforcement. Please remove when the feature is released to the public.
+    let organizationSettingsFieldsFiltered = organizationSettingsFields.slice();
+    if (!this.getFeatures().has('require-2fa')) {
+      organizationSettingsFieldsFiltered[2].fields.splice(0, 1);
```

Comment:
I would instead use `filter` and filter by field name, just in case the `fields` array changes.